odl/tomo/operators/ray_trafo.py

```diff
@@ -146,6 +149,21 @@ def __init__(self, reco_space, geometry, variant, **kwargs):
             if geometry.ndim > 2 and impl.endswith('cpu'):
                 raise ValueError('`impl` {!r}, only works for 2d'
                                  ''.format(impl_in))
+
+            # Filter axis-alinged initial detector position in parallel 3d
+            # geometry since it's broken for some ASTRA versions
+            if isinstance(geometry, Parallel3dAxisGeometry):
+                axis = geometry.axis
+                init_vec = (geometry._det_init_pos /
+                            np.linalg.norm(geometry._det_init_pos))
+                # Almost the same
+                if (np.isclose(np.dot(axis, init_vec), 1) and
+                        not astra_supports('par3d_axis_aligned_det_init_pos')):
```

Comment:
I do not, but we can reverse engineer it via the fix: 

https://github.com/astra-toolbox/astra-toolbox/commit/86ed34e9a5fa408c9338cd5c2871f7f7953806b7